### PR TITLE
feat(svg): Add usage note on deprecation of data URIs in use href attr

### PR DESCRIPTION
**Pull request by bsmth**
For good measure, adding a warning box to `<use>` about these cases:

```html
<svg>
  <use href="data:image/svg+xml;base64,PHN2ZyB4bWxucz0i...#x" />
</svg>
```

```html
<use>
  <set attributeName="href" to="data:image/svg+xml;base64,PHN2ZyB4bWxuc..."/>
</use>
```

```js
use.setAttribute("href", "data:image/svg+xml;base64,PHN2ZyB4...
```

__Related issues and pull requests:__

- [x] https://github.com/mdn/content/pull/31124

### CODE REVIEW

File: files/en-us/web/svg/element/use/index.md [hunk 1]
```diff
@@ -78,9 +78,13 @@ Since the cloned nodes are not exposed, care must be taken when using [CSS](/en-
 
 For security reasons, browsers may apply the [same-origin policy](/en-US/docs/Web/Security/Same-origin_policy) on `use` elements and may refuse to load a cross-origin URL in the {{SVGAttr("href")}} attribute. There is currently no defined way to set a cross-origin policy for `use` elements.
 
+> **Warning:** Loading resources with data URIs in the `href` attribute is deprecated for security reasons.
+> This applies to `<use href="data:...`, or setting `href` via [`<set>`](/en-US/docs/Web/SVG/Element/set) or [`setAttribute`](/en-US/docs/Web/API/Element/setAttribute).
+> See the [Browser Compatibility](#browser_compatibility) table for details.
+
```
Comment (dipikabh) [inline files/en-us/web/svg/element/use/index.md:84]:
```suggestion
> **Warning:** Loading resources with data URIs in the `href` attribute is deprecated for security reasons.
> This applies to `<use href="data:image/svg+xml;base64,PHN2Zucz0i...#x"` and also when setting `href` by using the [`<set>`](/en-US/docs/Web/SVG/Element/set) or [`setAttribute`](/en-US/docs/Web/API/Element/setAttribute) method.
> See the [Browser compatibility](#browser_compatibility) table for details.
```
- Borrowing the href example from your [release notes PR](https://github.com/mdn/content/pull/31124).
- Should we say which row in the bcd table to check --> "Load from data: URI"?
- I was wondering why we'd want readers to check bcd for this and whether this sentence is necessary here. Do we want to phrase it like:
> See "Load from data: URI" in the [Browser compatibility](#browser_compatibility) table to check its support across browsers over the years.

- Might be helpful for readers to also add a hint in `href` in the [Attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use#attributes) section, something like see "Usage notes" section for caveats

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @bsmth. I've added a few suggestions.

File: files/en-us/web/svg/element/use/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/svg/element/use/index.md]: Super, tnx. Added some changes in https://github.com/mdn/content/pull/31684/commits/18cdf779e53e720ab6ad7fca32dfceeeae2a0979

Re: `<use href="data:image/svg+xml;base64,PHN2Zucz0i`, you could use different schemes like `<use href="data:image/png` or maybe `<use href="data:something_else`. The Fx change is `data:...` being blocked, so I would stick with that: https://hg.mozilla.org/mozilla-central/rev/efcdf995dbc7#l2.16

File: files/en-us/web/svg/element/use/index.md [hunk 2]
```diff
@@ -78,9 +78,13 @@ Since the cloned nodes are not exposed, care must be taken when using [CSS](/en-
 
 For security reasons, browsers may apply the [same-origin policy](/en-US/docs/Web/Security/Same-origin_policy) on `use` elements and may refuse to load a cross-origin URL in the {{SVGAttr("href")}} attribute. There is currently no defined way to set a cross-origin policy for `use` elements.
 
+> **Warning:** Loading resources with data URIs in the `href` attribute is deprecated for security reasons.
+> This applies to `<use href="data:...` and also when setting `href` by using the [`<set>`](/en-US/docs/Web/SVG/Element/set) or [`setAttribute`](/en-US/docs/Web/API/Element/setAttribute) method.
```
Comment (dipikabh) [inline files/en-us/web/svg/element/use/index.md:82]:
```suggestion
> This applies to `<use href="data:...` and also when setting `href` by using the [`set`](/en-US/docs/Web/SVG/Element/set) or [`setAttribute`](/en-US/docs/Web/API/Element/setAttribute) method.
```

**Review by dipikabh — APPROVED** [review]
Just a tiny nit with the angle brackets around `set`, but leaving my +1.
Thank you for the updates!

**Comment by bsmth** [comment]
Super, thank you!

**Pull request merged**